### PR TITLE
Fix 404 layout

### LINKED ISSUE #43: Fix 404 layout
Probably related to floats, search box also needs work

| Desktop | Tablet | Mobile |
|-|-|-|
| wp20 mystagingwebsite com_blah(Desktop) | wp20 mystagingwebsite com_blah(iPad) | wp20 mystagingwebsite com_blah(iPhone X) |

### PR DESCRIPTION
**Pull request by adamwoodnz**
Closes #43 

| Desktop | Tablet | Mobile |
|-|-|-|
| localhost_8888_blah(Desktop) | localhost_8888_blah(iPad) | localhost_8888_blah(iPhone X) (1) |

### CODE REVIEW

File: wp-content/themes/twentyseventeen-wp20/style.css [hunk 1]
```diff
@@ -1650,3 +1654,44 @@ iframe.wp-embedded-content {
...
+	background-color: unset;
+}
+
+.error404 input[type="search"],
+.error404 .search-form .search-submit {
+	padding: var(--wp20--spacing--xxsmall) var(--wp20--spacing--xsmall);
+}
+
+.error404 .search-form .search-submit .icon {
+	display: none;
```
Comment (adamwoodnz) [inline wp-content/themes/twentyseventeen-wp20/style.css:1690]: Hide the 2017 icon

Comment (adamwoodnz) [inline wp-content/themes/twentyseventeen-wp20/style.css:1690]: Hide the 2017 icon

File: wp-content/themes/twentyseventeen-wp20/style.css [hunk 2]
```diff
@@ -779,11 +779,8 @@ body:not(.title-tagline-hidden) .site-branding-text {
 	padding: 0;
 }
 
-.navigation-top-menu-container .select2-search--dropdown .select2-search__field {
-	padding: 13px 50px 13px 12px;
-	border-radius: 0;
-	border: 0 !important;
-	border-bottom: 1px solid var(--wp20--color--border) !important;
+.navigation-top-menu-container .select2-search--dropdown .select2-search__field,
+.error404 .search-form .search-submit {
```
Comment (adamwoodnz) [inline wp-content/themes/twentyseventeen-wp20/style.css:783]: Use the same icon for the 404 search form as we do for the locale search

**Review by iandunn — APPROVED** [review]
LGTM 👍🏻

**Comment by StevenDufresne** [comment]
Looking at the screenshots, for the desktop view, can we either align it to the left with the menu/footer or center it?

**Comment by adamwoodnz** [comment]
> Looking at the screenshots, for the desktop view, can we either align it to the left with the menu/footer or center it?

@marko-srb what would you prefer here, I chose the constrained layout (used for News) due to the minimal content, thinking we only use the full width layout for pages with lots of content, like Swag, What's on, Live, etc.

**Comment by adamwoodnz** [comment]
Merging this as is and will iterate on layout if required.

**Pull request merged**